certtostore_windows.go

```diff
@@ -712,9 +718,9 @@ func (w *WinCertStore) remove(issuer string, removeSystem bool) error {
 	return nil
 }
 
-// removeCert wraps CertDeleteCertificateFromStore. If the call succeeds, nil is returned, otherwise
+// RemoveCertByContext wraps CertDeleteCertificateFromStore. If the call succeeds, nil is returned, otherwise
 // the extended error is returned.
-func removeCert(certContext *windows.CertContext) error {
+func (WinCertStore) RemoveCertByContext(certContext *windows.CertContext) error {
```

Comment:
Value receiver seems inconsistent with the rest of the WinCertStore methods. I'm not sure this needs to be a member of WinCertStore at all given the wrapped function has no dependencies other than the cert context, although it's not a deal breaker either way. I'd prefer Pointer receiver if we leave it this way though.

Reply:
Thank Matt for the review, I updated the code and converted this method a package function (not a member of WinCertStore)